prow/config/org/org.go

```diff
@@ -62,6 +63,21 @@ type Team struct {
 	Previously []string `json:"previously,omitempty"`
 }
 
+// Repo declares metadata about who has access to a repo.
+// Members denotes members of the organization with direct collaborator access.
+type Repo struct {
+	Members              UserPermissionLevel `json:"members,omitempty"`
```

Comment:
I don't understand this field, should it be map[string]UserPermissionLevel?


However, I would think that the better way to write this would be something like:

```golang
type Repo struct {
  Admins []string
  Pushers []string
  Pullers []string
}
```

So then I can do:
```yaml
orgs:
- kubernetes:
    repos:
    - test-infra:
        admins: ["test-infra-admins", "phippy"]
        writers: ["test-infra-writers", "goldie"]
        members: ["test-infra-members", "captain kube"]
```